main.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+let operators = document.querySelectorAll(".operator"),
+  numbers = document.querySelectorAll(".number"),
+  display = document.querySelector(".display"),
+  clearButtons = document.querySelectorAll(".cancel"),
+  result = document.querySelector("#result"),
+  point = document.querySelector("#point"),
+  a = "",
```

Comment:
a, b, c names doesn't tell me the function of this variables :)

Reply:
done)